__tests__/app.test.js

```diff
@@ -65,10 +65,86 @@ describe("GET /api/articles/:article_id", () => {
   test("requests for invalid article IDs are rejected", () => {
     return request(app)
       .get("/api/articles/999999")
-      .expect(404)
+      .expect(400)
```

Comment:
GET /api/articles/99999999 wouldn't be a bad request - it is valid - it just so happens that particular id doesn't exist